component/src/main/java/io/siddhi/extension/store/rdbms/RDBMSEventTable.java

```diff
@@ -637,7 +638,7 @@ protected void add(List<Object[]> records) throws ConnectionUnavailableException
         } catch (ConnectionUnavailableException e) {
             throw new ConnectionUnavailableException("Failed to add records to store: '" + this.tableName + "'", e);
         } catch (RDBMSTableException e) {
-            throw new RDBMSTableException("Failed to add records to store: '" + this.tableName + "'", e);
+            throw new DatabaseRuntimeException("Failed to add records to store: '" + this.tableName + "'", e);
```

Comment:
Shall we add the query too to the exception

Reply:
We cannot get the query at this point, hence it won't be possible